R/step-subset-separate.R

```diff
@@ -50,8 +50,7 @@ separate.dtplyr_step <- function(data, col, into,
     abort("`sep` must be a character vector.")
   }
 
-  sim_data <- simulate_vars(data)
-  col <- syms(names(tidyselect::eval_select(enquo(col), sim_data)))[[1]]
+  col <- as.symbol(tidyselect::vars_pull(data$vars, !!enquo(col)))
```

Comment:
Maybe use `sym()` instead of `as.symbol()` to stick with `rlang` conventions

Reply:
Not sure quite what happened here - looks like I both approved a different time and also left this comment

Reply:
We should be fine, I don't think it's worth the extra effort to fix.

Reply:
ah, guess we're both in a hurry to merge today, missed this. It's pretty minor so I just made a commit directly to main

Reply:
Sounds good. I had definitely messed something up in my review. I approved two separate times somehow. Once without a comment and once with a comment. Oh well.